_posts/en/newsletters/2022-12-07-newsletter.md

```diff
@@ -0,0 +1,153 @@
+
+- [BTCPay 1.7.1][] is the latest release of the most widely used
+  self-hosted payment processing software for Bitcoin.
+
+- [Core Lightning 22.11][] is the next major version of CLN.  It's also
+  be the first release to use a new version numbering scheme.[^semver]
+  Included are several new features, including a new plugin manager,
+  and multiple bug fixes.
+
+- [LND 0.15.5-beta][] is a maintenance release of LND.  It contains
+  only minor bug fixes according to its release notes.
+
+- [BDK 0.25.0-rc.1][] is a release candidate for this library for
+  building wallets.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19762][] updates the RPC (and, by extension,
+  bitcoin-cli) interface to allow named and positional arguments to be
+used together. This change makes it more convenient to use named
+parameter values without having to name every one.
+
+- [Core Lightning #5722][] adds [documentation][grpc doc] about how to
+  use the GRPC interface plugin.
+
+- [Eclair #2513][] updates how it uses the Bitcoin Core wallet to ensure
+  it always uses a [bech32][topic bech32] address (segwit output).  This
+  is the result of [Bitcoin Core #23789][] (see [Newsletter
+  #181][news181 bcc23789]) where the project addressed a privacy
+  concern for adopters of new output types (e.g. [taproot][topic
+  taproot]).  Previously, a user who set their wallet default address
+  type to taproot would also create taproot change outputs when they
+  paid someone.  If they paid someone who didn't use taproot, it was
+  easy for third parties to determine which output was the payment (the
+  non-taproot output) and which was the change output (the taproot
+  output).  After the change to Bitcoin Core, it would default to trying
+  to use the same type of change output as the paid output, e.g. a
+  payment to a native segwit output would also result in a native segwit
+  change output.
+
+    However, the LN protocol requires certain output types.  For
+    example, a P2WPKH output can't be used to open an LN channel.
+    For that reason, users of Eclair with Bitcoin Core need to ensure
+    they don't generate change outputs of an LN-incompatible type.
+
+- [Rust Bitcoin #1415][] begins using the [Kani Rust Verifier][] to
+  prove some properties of Rust Bitcoin's code.  This complements other
+  continuous integration tests performed on the code, such as fuzzing.
+
+- [BTCPay Server #4238][] adds an invoice refund endpoint to BTCPay's
+  Greenfield API, a more recent API different from the original Bitpay-inspired API.
```

Comment:
(not that I care too much about the correct spelling of a company made obsolete by the very software discussed in context)
```suggestion
  Greenfield API, a more recent API different from the original BitPay-inspired API.
```